Make linux-uclibcgnueabi an alias for linux-uclibceabi since it's
what we use in buildroot toolchains.
Slightly tweak musleabi to match the buildroot-used tuple.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura dpkg-1.18.3.orig/ostable dpkg-1.18.3/ostable
--- dpkg-1.18.3.orig/ostable	2015-10-15 08:05:04.471762555 -0300
+++ dpkg-1.18.3/ostable	2015-10-15 08:11:05.206132648 -0300
@@ -12,10 +12,10 @@
 # system part of the output of the GNU config.guess script.
 #
 # <Debian name>		<GNU name>		<config.guess regex>
-uclibceabi-linux	linux-uclibceabi	linux[^-]*-uclibceabi
+uclibceabi-linux	linux-uclibceabi	linux[^-]*-uclibc.*eabi
 uclibc-linux		linux-uclibc		linux[^-]*-uclibc
 musleabihf-linux	linux-musleabihf	linux[^-]*-musleabihf
-musl-linux		linux-musl		linux[^-]*-musl
+musl-linux		linux-musl		linux[^-]*-musl[^-]*
 gnueabihf-linux		linux-gnueabihf		linux[^-]*-gnueabihf
 gnueabi-linux		linux-gnueabi		linux[^-]*-gnueabi
 gnuabin32-linux		linux-gnuabin32		linux[^-]*-gnuabin32
@@ -34,6 +34,6 @@
 bsd-netbsd		netbsd			netbsd[^-]*
 bsd-openbsd		openbsd			openbsd[^-]*
 sysv-solaris		solaris			solaris[^-]*
-uclibceabi-uclinux	uclinux-uclibceabi	uclinux[^-]*-uclibceabi
+uclibceabi-uclinux	uclinux-uclibceabi	uclinux[^-]*-uclibc.*eabi
 uclibc-uclinux		uclinux-uclibc		uclinux[^-]*(-uclibc.*)?
 tos-mint		mint			mint[^-]*
